### ISSUE TITLE
fix Tester to match fixes in Trainer, i.e. tf.nn.softmax on predictions

### ISSUE THREAD

**Issue by NickleDave** [NickleDave/thrillington#27]

**Comment by NickleDave** [comment]
Not quite sure what I meant here but Tester works like Trainer at least as far back as  2c3643e2beaeaa9a1c8bfb6c6f7b7966e24f4205
